### PR TITLE
feat: Show add to calendar section for customers after booking an event 

### LINKED ISSUE #17749: Show "Add to calendar" on booking confirmation page for non-host users
### Is your proposal related to a problem?

When I am logged into my Cal.com account and use my own meeting link, I see an awesome section "Add to calendar" on the booking confirmation page:

Screenshot

This section is not visible when someone else uses my booking link (e.g. a customer).

However, having a way for my customers to add an event to their calendars from the booking page could dramatically help prevent no shows.

When the event is only sent via email, some people forget to add it to their calendars.

### Describe the solution you'd like

I'd love it if my customers saw the same "Add to calendar" section after booking an event with me.

That would remind them to make sure the event makes it onto their calendars.

### Describe alternatives you've considered

Currently, attendees have to go into their email inbox and add the event to their calendar.

Sometimes people forget, and then they don't show up to meetings.

### Additional context

This is the condition controlling the section:

https://github.com/calcom/cal.com/blob/7c5292e13beeb74f2488306198dc53a69620eccf/apps/web/modules/bookings/views/bookings-single-view.tsx#L792-L796

I know that you show a "Create your own booking link with Cal.com" section in the same place if the host hasn't disabled Cal.com branding.

So for me it would be fine if this section only showed up for paid accounts that have Cal.com branding disabled. (I'm happy to pay just for this feature!) Alternatively, you could show both sections.

### Requirement/Document

None

---

##### House rules

- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.
  - Follow Best Practices lined out in our [Contributor Docs](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by coder-adnan**
## What does this PR do?

This PR modifies the booking confirmation page to show the "Add to calendar" section for customers after booking an event. This change aims to help prevent no-shows by making it easier for customers to add the event to their calendars.

- Fixes #17749 

## Mandatory Tasks 

- [yes] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [yes] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

The test performed successful to show the 

This PR modifies the booking confirmation page to show the "Add to calendar" section for customers after booking an event. This change aims to help prevent no-shows by making it easier for customers to add the event to their calendars.

### CODE REVIEW

**Comment by coder-adnan** [comment]
This pull request fixes the issue of showing the "Add to calendar" section on the booking confirmation page for non-host users. Previously, this section was only visible to the host, which could lead to customers forgetting to add the event to their calendars and potentially missing the meeting.

The changes made include modifying the condition in the bookings-single-view.tsx file to ensure that the "Add to calendar" section is displayed for customers as well. This will help reduce no-shows by making it easier for customers to add the event to their calendars directly from the confirmation page.

Additionally, test cases in the [bookings-single-view.test.tsx](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) file have been updated to cover this new functionality, ensuring that the section is correctly displayed and functional for customers.

This fix addresses GitHub issue #17749.

File: apps/web/modules/bookings/views/bookings-single-view.tsx [hunk 1]
```diff
@@ -202,7 +202,8 @@ export default function Success(props: PageProps) {
   };
 
   function setIsCancellationMode(value: boolean) {
-    const _searchParams = new URLSearchParams(searchParams ?? undefined);
+    // Convert ReadonlyURLSearchParams to URLSearchParams
```
Comment (Praashh) [inline apps/web/modules/bookings/views/bookings-single-view.tsx:205]: remove comment.

**Comment by Praashh** [comment]
Hey @coder-adnan please Sign CLA

**Review by Praashh — COMMENTED** [review]
Hey @coder-adnan, can you please attach a screen recording of the fix?

**Review by Praashh — APPROVED** [review]
LGTM!!

**Pull request merged**